src/components/HubSpotForm.module.css

```diff
@@ -1052,3 +1044,20 @@
 .container p {
     margin-bottom: 10px !important;
 }
+
+.overrideFormShorten :global(.hs-firstname),
+.overrideFormShorten :global(.hs-lastname),
+.overrideFormShorten :global(.hs-phone),
+.overrideFormShorten :global(.hs-jobtitle),
+.overrideFormShorten :global(.hs-company),
+.overrideFormShorten :global(.hs-website),
+.overrideFormShorten :global(.hs-industry),
+.overrideFormShorten :global(.hs-company_size),
+.overrideFormShorten :global(.hs-annualrevenue),
+.overrideFormShorten :global(.hs-address),
+.overrideFormShorten :global(.hs-city),
+.overrideFormShorten :global(.hs-state),
+.overrideFormShorten :global(.hs-country),
+.overrideFormShorten :global(.hs-zip) {
+    display: block !important;
+}
```

Comment:
This `overrideFormShorten` class and prop overrides `display: none` style rule made by ClearBit 

<img width="1496" alt="image" src="https://github.com/sourcegraph/about/assets/51897872/3f12809e-eb42-4694-a4d2-17f98f1bf61d">